### PR TITLE
Update material-ui

### PR DESCRIPTION
**Pull request by ShirasawaSama**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] README update
- [x] Other (about what?)

### 🔗 Related issue link

#1 

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Demo in storybook is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] README API section is updated or not needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,50 +34,47 @@
...
-    "@types/node": "^14.14.20",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.12.0",
-    "@typescript-eslint/parser": "^4.12.0",
+    "@babel/cli": "^7.14.5",
+    "@babel/core": "^7.14.6",
+    "@babel/plugin-proposal-export-default-from": "^7.14.5",
+    "@babel/plugin-transform-typescript": "^7.14.6",
+    "@emotion/react": "^11.4.0",
```
Comment (baymac) [inline package.json:47]: Why is emotion added as a dep?

**Review by baymac — COMMENTED** [review]
Apart from the comment above, everything looks good to me. I will find some time today to test these changes.

File: package.json (hunk 1 above)
Comment (ShirasawaSama) [reply package.json:47]: It is a dependency of the new version of `material-ui`

Comment (ShirasawaSama) [reply package.json:47]: In order to support the work of the sample storybook page.

**Review by baymac — APPROVED** [review]
Lgtm! Thanks for your contribution.

**Comment by baymac** [comment]
Just one suggestion, send pr from a separate branch instead of master else it gets difficult to pull the changes.

**Pull request merged**